bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/main/java/com/google/cloud/bigtable/hbase2_x/BigtableAdmin.java

```diff
@@ -360,15 +361,34 @@ public void deleteColumnFamily(TableName tableName, byte[] columnName) throws IO
   }
 
   @Override
-  public void modifyTable(TableDescriptor arg0) throws IOException {
-    throw new UnsupportedOperationException("modifyTable"); // TODO
+  public void modifyTable(TableDescriptor tableDescriptor) throws IOException {
+    modifyTable(tableDescriptor.getTableName(), tableDescriptor);
   }
 
   @Override
-  public void modifyTable(TableName arg0, TableDescriptor arg1) throws IOException {
-    throw new UnsupportedOperationException("modifyTable"); // TODO
+  public void modifyTable(TableName tableName, TableDescriptor tableDescriptor) throws IOException {
+    System.out.println("********************* IN modifyTable ***********************");
+    if (isTableAvailable(tableName)) {
+      TableDescriptor currentTableDescriptor = getTableDescriptor(tableName);
+      List<Modification> modifications = new ArrayList<>();
+      List<HColumnDescriptor> columnDescriptors = toHColumnDescriptors(tableDescriptor);
+      List<HColumnDescriptor> currentColumnDescriptors = toHColumnDescriptors(currentTableDescriptor);
+      modifications.addAll(tableModificationAdapter.buildModificationsForModifyFamilies(columnDescriptors, currentColumnDescriptors));
```

Comment:
Please combine the two calls into the adapter into a single one that performs the same ling:  `tableModificationAdapter.buildModifications(columnDescriptors, currentColumnDescriptors)`